css-flexbox-photo-gallery/css/style.css

```diff
@@ -0,0 +1,15 @@
+* {
```

Comment:
I believe this is the first CSS file, so it can be renamed to something like `style00.css`. Once the HTML is at this point, you can start doing the basic styles listed in this file:

```
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>CSS Flex Photo Gallery</title>
    <link rel="stylesheet" href="./css/style50.css">
</head>
<body>
    <!-- Header -->
    <div class="header">
        <h1>CSS FLEXBOX PHOTO GALLERY</h1>
    </div>
</body>
</html>
```

Learners should be writing the code line by line, so this first CSS file could be completely empty with the instructions:

```
/*
Create a new rule using the universal `*` selector to target all elements on the page and set the `box-sizing` property to `border-box`.
This will make it so borders do not add to the size of each element on the page, and make calculating their actual size easier.
*/
```